### PR TITLE
[READY] blog performance support

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Blog from Joep about Performance Support for today. 

Text is already checked by Iwanna. 
Just scan it please, and check if technical everything is correct.

### CODE REVIEW

File: source/blog/nl/2016-07-05-bevoegd-niet-bekwaam-waar-bekwaamheid-over-gaat.html.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+title: Bevoegd is niet bekwaam – Waar bekwaamheid echt over gaat
+date: 2016-07-05 08:00 CEST
+tags:
+author: Joep
+lang: nl
+featured: true
+image: images/social/bevoegd-is-niet-bewkaam-1200x630.png
+---
+
+**Aantoonbaar maken dat je personeel bekwaam is. Dat is de vraag die vanuit accrediterende instanties en de Inspectie voor Gezondheidszorg (IGZ) aan de zorginstellingen in Nederland gesteld wordt. Het is de vraag die voor mij als consultant van Defacto brood op de plank verzorgt. Maar aantoonbaar bekwaam, wat is dat nou eigenlijk? En hoe krijg je aantoonbare bekwaamheid voor elkaar?**
```
Comment (fatboypunk) [inline source/blog/nl/2016-07-05-bevoegd-niet-bekwaam-waar-bekwaamheid-over-gaat.html.markdown]: `Dat is de vraag` ->  `Dat is de eis`?
it's not really a question

Comment (fatboypunk) [inline source/blog/nl/2016-07-05-bevoegd-niet-bekwaam-waar-bekwaamheid-over-gaat.html.markdown]: First time `Defacto` could be a link?

Comment (IwannadeJonge) [inline source/blog/nl/2016-07-05-bevoegd-niet-bekwaam-waar-bekwaamheid-over-gaat.html.markdown]: Hi Marcel, I agree with your comments. Can you correct them? Thanks!

Comment (RonaldDijkstra) [inline source/blog/nl/2016-07-05-bevoegd-niet-bekwaam-waar-bekwaamheid-over-gaat.html.markdown]: I'll do this.

**Pull request merged**